selfdrive/car/toyota/carcontroller.py

```diff
@@ -64,17 +64,17 @@ def update(self, enabled, CS, frame, actuators, pcm_cancel_cmd, hud_alert,
 
     # gas and brake
     apply_gas = 0.
-    apply_accel = actuators.gas - actuators.brake
+    apply_accel = (actuators.gas - actuators.brake) * CarControllerParams.ACCEL_SCALE
 
     if CS.CP.enableGasInterceptor and enabled and CS.out.vEgo < MIN_ACC_SPEED:
       # converts desired acceleration to gas percentage for pedal
       # +0.06 offset to reduce ABS pump usage when applying very small gas
-      if apply_accel * CarControllerParams.ACCEL_SCALE > coast_accel(CS.out.vEgo):
-        apply_gas = clip(compute_gb_pedal(apply_accel * CarControllerParams.ACCEL_SCALE, CS.out.vEgo), 0., 1.)
-      apply_accel = 0.06 - actuators.brake
+      apply_gas = compute_gb_pedal(apply_accel, CS.out.vEgo, CS.out.brakeLights)
+      if apply_accel > 0 and CS.out.vEgo <= CS.CP.minSpeedCan:  # increase accel to release brake quicker todo: might not be needed, test with and without in similar situations
+        apply_accel *= 2
 
-    apply_accel, self.accel_steady = accel_hysteresis(apply_accel, self.accel_steady, enabled)
-    apply_accel = clip(apply_accel * CarControllerParams.ACCEL_SCALE, CarControllerParams.ACCEL_MIN, CarControllerParams.ACCEL_MAX)
+    apply_accel = clip(apply_accel, CarControllerParams.ACCEL_MIN, CarControllerParams.ACCEL_MAX) if enabled else 0.
+    apply_gas = clip(apply_gas, 0., 1.)
```

Comment:
Finally I removed hysteresis since it doesn't really make much since as far as I understand and added a final `if enabled` to just make sure it never sends non zero if not engaged